sw/host/hsmtool/commands/util.py

```diff
@@ -0,0 +1,189 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import datetime
+import io
+import json
+import random
+import tarfile
+import time
+
+import pkcs11
+
+
+class KeyNotFoundError(Exception):
+    pass
+
+
+class ContentError(Exception):
+    pass
+
+
+def ParseBool(b):
```

Comment:
https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
Or add a note in the package docstring describing the convention for this module.